source/server/LambdaLocationUpdateHandler.js

```diff
@@ -42,21 +42,19 @@ exports.handler = (event, context, callback) => {
         "Item" : locationItem
     }, function(error, data) {
         if (error) {
-            abortLocationUpdate("ERROR: INSERTING ITEM INTO DYNAMODB FAILED: " + error, callback);
+            abortLocationUpdate("Inserting location update into DynamoDB failed: " + error, callback);
         } else {
             callback(null, {
-                "statusCode" : 200,
-                "headers" : { "Content-Type" : "text/html" },
```

Comment:
`"Content-Type"` should be `application/json` 😄

Reply:
Ah yes, but this isn't the latest commit. In the latest one we don't use `statusCode`s or `headers`

Reply:
Ah sorry about that. The `statusCode` in this example is a HTTP header though so should it not be added to the latest commit?

i.e.

    callback(null, {
        "statusCode" : 200,
        "headers" : { "Content-Type" : "application/json" },
        "body" : {
            "status" : "error (or whatever)",
            "message" : "your message"
        }
    }

I might be wrong but it just seems strange that we aren't sending back an HTTP 200 successful repsonse code?

Reply:
Ahhh okay, I shall change it to that. I'm sure you know a lot more than me about HTTP requests/responses!

Reply:
Sorry for being picky!!! We'll test it once you've made changes and then we can merge the PR once it is tested!

Reply:
Picky will get us marks 🙏 👍